assets/js/blocks/cart-checkout/checkout-i2/inner-blocks/checkout-actions-block/index.tsx

```diff
@@ -21,6 +21,12 @@ registerFeaturePluginBlockType( 'woocommerce/checkout-actions-block', {
 		align: false,
 		html: false,
 		multiple: false,
+		reusable: false,
+		inserter: false,
```

Comment:
Removes the block from this list
![image](https://user-images.githubusercontent.com/6165348/124760983-36bde580-df29-11eb-83ca-06b5074ebc38.png)